model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -439,6 +439,12 @@ object WorkspaceType {
       case _              => throw new RawlsException(s"Invalid WorkspaceType [$wt]")
     }
 
+  def toCloudPlatform(t: WorkspaceType): WorkspaceCloudPlatform.WorkspaceCloudPlatform =
```

Comment:
I'm not sure if we want to bake this assumption in, as we will use McWorkspaces for GCP before too long. I think the McWorkspace has a cloud context/platform variable that can be checked (and RawlsWorkspace does map to GCP).

Reply:
I'm not too sure what the whole "multi-cloud" thing really means - maybe here's a good place to ask? No workspace is "multi-cloud" - it's either GCP- or Azure-hosted so what does multi-cloud really mean?

Reply:
Actually, I think I'm wrong. This is the code in WorkspaceServices' listWorkspaces where we get the cloudPlatform.

But it is a good question… I thought McWorkspace were "multi-cloud" because they could be either Azure or GCP. But I also recall discussions that someday a McWorkspace might have multiple billing profiles (for different clouds). @aherbst-broad can you explain what "multi-cloud" is meant to refer to?

```
                  val cloudPlatform = workspace.workspaceType match {
                    case WorkspaceType.McWorkspace =>
                      Option(workspaceManagerDAO.getWorkspace(workspace.workspaceIdAsUUID, ctx)) match {
                        case Some(mcWorkspace) if mcWorkspace.getAzureContext != null =>
                          Option(WorkspaceCloudPlatform.Azure)
                        case Some(mcWorkspace) if mcWorkspace.getGcpContext != null =>
                          Option(WorkspaceCloudPlatform.Gcp)
                        case _ =>
                          throw new RawlsException(
                            s"unexpected state, no cloud context found for workspace ${workspace.workspaceId}"
                          )
                      }

Reply:
So, "McWorkspace" was a term I introduced in the early going that I somewhat regret. It was in reference to the "MC Terra Architecture", a term which has been floating around to mean "not rawls". 

What it has really come to mean, though, is that this is a WSM-owned workspace rather than a legacy rawls workspace. It's defacto azure-only today; someday, as we embark on that migration, MC workspaces will also support GCP. 

To add to the potential complexity, a workspace manager workspace _does_ support multiple cloud contexts (i.e, you can have one GCP cloud context, and one Azure cloud context), but that is not something we will be leveraging any time soon.